src/devtools/client/debugger/src/reducers/ui.js

```diff
@@ -14,6 +14,8 @@ import { prefs, features } from "../utils/prefs";
 export const createUIState = () => ({
   selectedPrimaryPaneTab: "sources",
   activeSearch: null,
+  fullTextQuery: "",
```

Comment:
Nit but probably best to just use the full name here so `fullTextSearchQuery` and `fullTextSearchFocus` so searching for references is easier.